fix(core): Use safe_block_on in mobile proxy (#12514)
tauri-apps/tauri: .changes/fix-core-ios-async-panic.md

---
tauri: "patch:bug"
---

> Fixed an issue that caused iOS apps to panic with an async `run()` entrypoint.

Reviewer: ```suggestion
Fixed an issue that caused iOS apps to panic when using an async function for `tauri::mobile_entry_point`.
```

Author: I abused my powers in this repo once (to hotfix the cli, just squeezed this one in) and then this x) I'll fix it in the covector pr.
